generate_trajectories.py

```diff
@@ -19,8 +18,10 @@
 parser.add_argument('-data_dir', type=str, default='traffic-data/state-action-cost/')
 parser.add_argument('-fps', type=int, default=30)
 parser.add_argument('-time_slot', type=int, default=0)
-parser.add_argument('-map', type=str, default='i80', choices={'ai', 'i80', 'us101', 'lanker', 'peach'})
+parser.add_argument('-map', type=str, default='i80', choices={'ai', 'i80', 'us101', 'lanker', 'peach', 'highD'})
 parser.add_argument('-delta_t', type=float, default=0.1)
+parser.add_argument('-recording', type=str, default="01",
```

Comment:
Can't we use `time_slot` instead?

Reply:
Good point. I think this is an artifact of me trying to keep this new script as similar to map_i80 as possible

Reply:
Let's use `time_slot` for `highd` too.